Docker/README.md

```diff
@@ -79,24 +101,22 @@ For running the analysis, start the container (e.g. to run segmentation on __all
 ```bash
 docker run --gpus all -v /home/user/my_mri_data:/data \
                       -v /home/user/my_fastsurferCNN_analysis:/output \
-                      --rm --user 123X fastsurfercnn:gpu \
-                      --i_dir /data \
-                      --in_name mri/orig.mgz \
-                      --o_dir /output \
-                      --out_name mri/aparc.DKTatlas+aseg.deep.mgz \
-                      --log deep_surfer.log
+                      --rm --user $(id -u):$(id -g) fastsurfercnn:gpu \
```

Comment:
Might make sense to keep this consistent, i.e. deepmi/fastsurfer:gpu-segonly-v2.0.0. @m-reuter what do you think?

Reply:
Should be updated akin to line 81